### ISSUE TITLE
Turning on the File Caching With ION (Not mentioned nowhere)

### ISSUE THREAD

**Issue by vladimirarevshatyan** [koush/ion#815]
How do i 'Turn on' the file caching instead of memory caching?
I do not want to reload the image everytime the app is closed so the File caching is what i am willing, so do you support that? 
If yes,do you handle the runtime permission of Marshamllow and above?
Please answer

**Comment by ziobudda** [comment]
Any cache ?

**Comment by vladimirarevshatyan** [comment]
Yup, there is a cache on the Application level , probably LRU cache or something like that, ION caches them during the application lifecycle, but when app is killed and opened again the cache ia gone, so thats why I reckon it is LRU memory caching.
